tests/containers/bci_prepare.pm

```diff
@@ -41,6 +41,8 @@ sub packages_to_install {
         my $version = "$version.$sp";
         push @packages, 'python3-devel';
         if ($version eq "12.5") {
+            # PackageHub is needed for jq
+            script_retry("SUSEConnect -p PackageHub/12.5/$arch", delay => 60, retry => 3);
```

Comment:
Why not use the provided standard subroutines?

```suggestion
            add_suseconnect_product(get_addon_fullname('phub'));
```

Reply:
Open the VR and see the `BUILD` and `VERSION`.

Reply:
I see. Don't mind me resolving the issue here as well.